hipcheck/Cargo.toml

```diff
@@ -54,7 +53,7 @@ serde_derive = "1.0.137"
 serde_json = "1.0.119"
 smart-default = "0.7.1"
 spdx-rs = "0.5.0"
-termcolor = "1.1.3"
+# termcolor = "1.1.3"
```

Comment:
If we're not using it, I think it's better to remove fully rather than comment out.

Reply:
Fixed!

Reply:
Thanks!